### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ManosMpampis**
fixing outdated links

### CODE REVIEW

**Review by ad-daniel — APPROVED** [review]
Since it doesn't serve a purpose any longer, we can delete it. Thank you.

**Review by passalis — APPROVED** [review]
Thank you for the careful check of the docs and corrections! Seems fine to me.

File: docs/reference/engine-target.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ This target is used for keypoint detection in pose estimation, body part detecti
 A keypoint is a list with two coordinates [x, y], which gives the x, y position of the
 keypoints on the image.
 
-The [Keypoint](#class_engine.target.Keypoint) class has the following public methods:
+The [Keypoint](/src/opendr/engine/target.py#L192) class has the following public methods:
 #### Keypoint(keypoint, confidence=None)
-  Construct a new [Keypoint](#class_engine.target.Keypoint) object based from *keypoint*.
+  Construct a new Keypoint* object based from *keypoint*.
```
Comment (ad-daniel) [inline docs/reference/engine-target.md:56]:
```suggestion
  Construct a new *Keypoint* object based from *keypoint*.
```

File: docs/reference/engine-target.md [hunk 2]
```diff
@@ -118,22 +118,22 @@ Bases: `engine.target.Target`
...
 Additional fields are used to describe confidence (score), 2D projection of the box on camera image (bbox2d),
 truncation (truncated) and occlusion (occluded) levels, the name of an object (name) and
 observation angle of an object (alpha).
 
-The [BoundingBox3DList](#class_engine.target.BoundingBox3DList) class has the following public methods:
+The [BoundingBox3DList](/src/opendr/engine/target.py#L404) class has the following public methods:
 #### BoundingBox3DList(bounding_boxes_3d)
-  Construct a new [BoundingBox3DList](#class_engine.target.BoundingBox3DList) object based on the *bounding_boxes_3d*.
-  *bounding_boxes_3d* is expected to be a list of [BoundingBox3D](#class_engine.target.BoundingBox3D).
+  Construct a new BoundingBox3DList* object based on the *bounding_boxes_3d*.
```
Comment (ad-daniel) [inline docs/reference/engine-target.md:129]:
```suggestion
  Construct a new *BoundingBox3DList* object based on the *bounding_boxes_3d*.
```

File: docs/reference/index.md [hunk 3]
```diff
@@ -79,7 +79,47 @@ Neither the copyright holder nor any applicable licensor will be liable for any
     - [opendr_utils.h](c-opendr-utils-h.md)
     - [face_recognition.h](c-face-recognition-h.md)
 - Projects
+    - `c API` Module
```
Comment (ad-daniel) [inline docs/reference/index.md:82]:
```suggestion
    - `C API` Module
```

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Very nice, thank you very much for this effort!
I only have 3 small details to change

**Comment by passalis** [comment]
Thank you! I committed the suggested changes, since Manos will not be available today.

**Review by ad-daniel — APPROVED** [review]
Thank you!

**Review by passalis — APPROVED** [review]

**Pull request merged**